index.html

```diff
@@ -41,10 +41,20 @@ <h2>Why you should use it</h2>
 			</div>
 		</div>
 		<div class="row">
+			<div class="col-sm-4 feature">
+				<h3>Modern Features</h3>
+				<p>Push notifications, link previews, new message markers, and more bring IRC to the 21st century.</p>
```

Comment:
"new message markers" seems like such a small feature to advertise now 😅
Replace with "file upload" instead?